baseplate/experiments/targeting/tree_targeting.py

```diff
@@ -51,9 +51,12 @@ def __init__(self, input_node):
 
     def evaluate(self, **kwargs):
         candidate_value = kwargs.get(self._accepted_key)
-        if candidate_value in self._accepted_values:
-            return True
-
+        try:
+            if candidate_value in self._accepted_values:
+                return True
+        except TypeError:
+            pass
```

Comment:
Should this at least have a `logger.warn` or something similar so people don't wonder why their experiments are silently failing?